README.md

```diff
@@ -88,6 +68,35 @@ Facebook: `scope: "email,user_gender""`
 VK: `scope: "email"`  
 Google: `scope: "https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email"`  
  
+## Local build & serve
+The easiest way to run the application is running it in docker.
+Make sure you have docker installed. To build docker image run the following command from root of the project folder:  
+`docker build -f .docker/Dockerfile -t <image_name> .` Don't forget about the dot at the end of the command.  
+To serve the project locally run `docker run --name <container_name> -v </path/to/config.json>:/var/www/assets/config.json -d -p 80:80 <image_name>`  
```

Comment:
Once we templated everything, we might as well replace `-p 80:80` with something like `-p <port>:80`